src/components/Sparklines.js

```diff
@@ -0,0 +1,54 @@
+import { LineChart, Line } from 'recharts';
+
+const staticData = [
+  {
+    name: 'Page A',
+    uv: 4000,
+    pv: 2400,
+    amt: 2400,
+  },
+  {
+    name: 'Page B',
+    uv: 3000,
+    pv: 1398,
+    amt: 2210,
+  },
+  {
+    name: 'Page C',
+    uv: 2000,
+    pv: 9800,
+    amt: 2290,
+  },
+  {
+    name: 'Page D',
+    uv: 2780,
+    pv: 3908,
+    amt: 2000,
+  },
+  {
+    name: 'Page E',
+    uv: 1890,
+    pv: 4800,
+    amt: 2181,
+  },
+  {
+    name: 'Page F',
+    uv: 2390,
+    pv: 3800,
+    amt: 2500,
+  },
+  {
+    name: 'Page G',
+    uv: 3490,
+    pv: 4300,
+    amt: 2100,
+  },
+];
+
+export default function Sparkline() {
```

Comment:
The Sparkline component should be able to take in the data as a parameter. You can look at a lot of the other components in the app to get an idea of how that works (like the `SectionCard` is good example)

Reply:
The sparklines are currently receiving the data as a parameter, However it isn't currenlty flexable enough to change it's data souce depending on what row of the table it is on. I attempted to address this by using field.name as a way to pull from a object contatining diffrenct sets of data but I unfortunatly couldn't get this to work. I would love to hear if you have any suggestions on how I can get this to work!